lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -77,25 +78,41 @@ $description = Get-AnsibleParam -obj $params -name "description" -type "str" -de
 $enabled = Get-AnsibleParam -obj $params -name "enabled" -type "bool" -default $true
 $path = Get-AnsibleParam -obj $params -name "path" -type "str" -default '\'
 
+# Ensure the TaskPath starts with and ends with slashes as this causes issues with some
+# versions of Windows
+if ($path[0] -ne '\') {
```

Comment:
@dagwieers understood, I'll make the change.

I was trying to explain that I'm trying to apply the "be liberal in what you accept" philosophy.  In this case there isn't really ambiguity between having a slash and missing a slash.  Missing the slash at the beginning currently throws an ambiguous error because the Powershell command requires a backslash at the beginning of the path.